app/editor/src/features/content/form/ClipDirectoryTable.tsx

```diff
@@ -36,8 +36,8 @@ export const ClipDirectoryTable: React.FC<IClipDirectoryTableProps> = ({
   onAttach,
   navigate,
 }) => {
-  const { values } = useFormikContext<IItemModel>();
-
+  const { values } = useFormikContext<IItemModel>() ?? {};
```

Comment:
We should not do this.  All you're doing it hiding errors from the linter.

Reply:
Updated, thanks